src/Relation/Nullary/Decidable/Core.agda

```diff
@@ -52,6 +51,15 @@ open Dec public
 pattern yes p =  true because ofʸ  p
 pattern no ¬p = false because ofⁿ ¬p
 
+------------------------------------------------------------------------
+-- decision procedures are Decidable
+
+bool⁺ : ∀ b → Dec (T b)
+bool⁺ b = b because (T⁺ b)
+
+predᵇ⁺ : ∀ {a} {A : Set a} (p : A → Bool) (x : A) → Dec (T (p x))
+predᵇ⁺ p x = bool⁺ (p x)
```

Comment:
I'm unsure about how useful this is? Isn't much clearer to write `T? (p x)` rather than `predᵇ⁺ p x`? The former is shorter as well.

Reply:
Yes, I was going back on myself about this definition in any case. My (overly-fetishing-uniformity/consistency) brain was thinking that `T?` (as suggested above) from `Relation.Nullary.*` should be being systematically/successively lifted in some way to `Relation.Unary` and `Relation.Binary`... and this was a clumsy way of stumbling on that thought. Happy to suppress it!

Reply:
Have now removed `predᵇ⁺` (but with a thought to replicate it later, under the name `T?`, in `Relation.Unary`).